static/app/views/starfish/views/spanSummaryPage/spanTransactionsTable.tsx

```diff
@@ -50,10 +50,7 @@ type Row = {
 
 type Props = {
   sort: ValidSort;
-  span: Pick<
-    SpanMetricsFieldTypes,
-    SpanMetricsField.SPAN_GROUP | SpanMetricsField.SPAN_OP
-  >;
+  span: Pick<MetricsResponse, SpanMetricsField.SPAN_GROUP | SpanMetricsField.SPAN_OP>;
```

Comment:
`MetricsResponse` is probably a more descriptive way to describe the type anyway